src/new/wallet.js

```diff
@@ -325,15 +325,52 @@ class HathorWallet extends EventEmitter {
       }],
     };
 
+    return this.completeTxData(data, txToken);
+  }
+
+  /**
+   * Complete transaction data with inputs
+   *
+   * @param {Object} data Partial data that will be completed with inputs
+   * @param {Object} token Token object {'uid', 'name', 'symbol'}. Optional parameter if user already set on the class
+   *
+   * @return {Object} Object with 'success' and completed 'data' in case of success, and 'message' in case of error
+   **/
+  completeTxData(partialData, token) {
+    const txToken = token || this.token;
     const walletData = wallet.getWalletData();
     const historyTxs = 'historyTransactions' in walletData ? walletData.historyTransactions : {};
-    const ret = wallet.prepareSendTokensData(data, txToken, true, historyTxs, [txToken]);
+    const ret = wallet.prepareSendTokensData(partialData, txToken, true, historyTxs, [txToken]);
+
+    return ret
+  }
 
-    if (!ret.success) {
-      return ret;
+  /**
+   * Send a transaction from its outputs
+   *
+   * @param {Array} outputs Array of outputs with each element as an object with {'address', 'value'}
+   *
+   * @return {Promise} Promise that resolves when transaction is sent
+   **/
+  sendManyOutputsTransaction(outputs) {
+    storage.setStore(this.store);
+    const data = {
+      tokens: [], // For now does not support custom tokens
+      inputs: [],
+      outputs: [],
+    };
+
+    for (const output of outputs) {
+      data.outputs.push({address: output.address, value: output.value, tokenData: 0})
```

Comment:
It's always HTR?

Reply:
Yes. There is a comment above this line saying that

```
    const data = {
      tokens: [], // For now does not support custom tokens
      inputs: [],
      outputs: [],
    };
```

I started doing the multi token method but it's more complicated than that, so I decided to keep this way just to solve the problem we have now with Euroline. This code will completely change after the wallet service, so there is no reason to support that now.

Reply:
Got it, we just need to add this comment to the docstring.

Reply:
Done